api/Document.json

```
@@ -3103,7 +3103,10 @@
               {
                 "version_added": "16.4",
                 "partial_implementation": true,
-                "notes": "Only available on iPad, not on iPhone."
+                "notes": [
+                  "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled.",
```

To not say the same thing in two notes:

```suggestion
                  "Only available on iPad, not on iPhone.",
```

Can you make the same change throughout?

---

Oh, I'm sorry, I see now why you did this. I suggested it! That was a copy-paste-edit failure, sorry about that! I'll fix it.